Cargo.toml

```diff
@@ -43,7 +43,7 @@ num-bigint = "0.1.35"
 protobuf = "1.1"
 rand = "0.3.13"
 rpassword = "0.3.0"
-rust-crypto = { git = "https://github.com/awmath/rust-crypto.git", branch = "avx2" }
+rust-crypto = "0.2.36"
```

Comment:
This needs to be done in every crate, eg `core/Cargo.toml`.

Reply:
slipped my mind, will do that now.